Client/Frontend/Notifications/NotificationRootViewController.swift

```diff
@@ -88,8 +97,15 @@ extension NotificationRootViewController {
 
 // MARK: - Notification API
 extension NotificationRootViewController {
-    func showStatusNotification() {
+    func showStatusNotification(duration duration: NotificationDuration?) {
```

Comment:
> Would I have to dismiss the syncing... message, causing it to animate away, and then animate it back in with the new message for 4 seconds?

At the moment yea it seems that if 'Syncing...' is showing we would need to dismiss it and re-show it with the failed message causing the double animation. I can add an `animated` property to both methods and if we are showing a notification while another one is active we can hide/show animation to give it the appearance of updating.

Thoughts?